packages/wrangler/src/index.tsx

```diff
@@ -187,32 +187,52 @@ export async function main(argv: string[]): Promise<void> {
         type: "string",
       });
     },
-    async () => {
+    async (args) => {
+      if ("type" in args) {
+        let message =
+          "The --type option is no longer supported.\n" +
+          "The difference between 'javascript' and 'rust' is automatically inferred.";
```

Comment:
This line isn't actually true (they're actually both the same). We should have a link here to a doc that explains the nuance, but we don't have that yet. Let's just remove this line?
```suggestion
          "The --type option is no longer supported."; 
```

Reply:
Cool